Tests/CombineFeedbackTests/CombineFeedbackTests.swift

```diff
@@ -128,4 +128,58 @@ class CombineFeedbackTests: XCTestCase {
 
         XCTAssertEqual("initial_a", value)
     }
+  
+    func test_system_in_a_system() {
+        let scheduler = TestScheduler()
+      
+        let feedback1 = Feedback<String, String>(effects: { state -> AnyPublisher<String, Never> in
+          if state.count % 3 == 1 {
+              return Just("a").eraseToAnyPublisher()
```

Comment:
I would suggest to add `_a` so it's easier to understand what's happening in the assertion

Reply:
This is the best example I could write at the time that uses simple string manipulation to display a system within a system. The outer system sends the inner system a string, the inner system accepts the string and returns another string back to the outer system as an event. The Outer system then appends the string to it's state and then because we've reached a point in the inner systems where it will no longer send a new event. `(state.count % 3 == 2)` Is that state. It isn't obvious because it is not written.

Looking at it 20 days later I've got to ask myself if it is a good choice for an example, not really? But it certainly is okay for testing that things should work.  

However if I tried to keep your system of using `_` to operate the events my simple use of '%' will no longer work since state.count will increase in a different way.

Any other recommendations? I can go farther, not use a string as the state but and make the internal state model something else, but I'd like your approval on that stylistically first since it will no longer be a file of test that always uses a simple string as it's state